app/src/main/java/com/ikarabulut/GameDisplay.java

```diff
@@ -25,14 +26,32 @@ public GameDisplay() {
         rulesPrompt = "Which RuleSet would you like to play with?" + "\n"
                 + "1: Alive Cell dies with less than 2 or more than 3 alive neighbors. Dead Cell comes to life with exactly 3 alive neighbors" + "\n"
                 + "2: Alive Cell die with less than 2 or more than 3 alive neighbors. Dead Cell comes to life with exactly 4 alive neighbors" + "\n"
```

Comment:
Some of these prompts say "Alive Cell dies" and some say "Alive Cell die". Keep the wording consistent wherever possible.